service/csv/csv.go

```diff
@@ -126,13 +162,21 @@ func AddLine(f *os.File, lines [][]string, newJobs *[]model.ExtJob) error {
 	return nil
 }
 
-// OpenAndWrite function
-func OpenAndWrite(path string) (*os.File, error) {
-	f, err := os.OpenFile(path, os.O_RDONLY|os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+// GetJobs function
+func (s *CsvService) GetJobs() ([]model.Job, error) {
+	f, err := Open(pathFile)
+
 	if err != nil {
-		return nil, errors.New("There was an error opening the file")
+		return nil, err
 	}
-	return f, nil
+
+	jobs, errorReading := Read(f)
+
+	if errorReading != nil {
+		return nil, errorReading
+	}
+
+	return jobs, nil
```

Comment:
This could be simplified as:
```suggestion
	return Read(f)
```